### PR TITLE
fix: change apim rest api folder name (monorepo)

### PR DESCRIPTION
**Pull request by phiz71**
gravitee-io/issues#6092

### CODE REVIEW

File: src/main/docker/Dockerfile-build-full-ee-v3 [hunk 1]
```diff
@@ -13,9 +13,9 @@ ENV GRAVITEEIO_HOME=$GRAVITEEIO_HOME
...
+RUN mkdir -p ${GRAVITEEIO_HOME}/graviteeio-full-${APIM_VERSION}/graviteeio-apim-rest-api-${APIM_VERSION}/ && mkdir -p ${GRAVITEEIO_HOME}/graviteeio-full-${APIM_VERSION}/graviteeio-gateway-${APIM_VERSION}/
 # Prepare lib and plugins directories
-RUN mkdir -p ${GRAVITEEIO_HOME}/graviteeio-full-${APIM_VERSION}/graviteeio-rest-api-${APIM_VERSION}/lib/ && mkdir -p ${GRAVITEEIO_HOME}/graviteeio-full-${APIM_VERSION}/graviteeio-gateway-${APIM_VERSION}/lib/ && mkdir -p ${GRAVITEEIO_HOME}/graviteeio-full-${APIM_VERSION}/graviteeio-rest-api-${APIM_VERSION}/plugins/ && mkdir -p ${GRAVITEEIO_HOME}/graviteeio-full-${APIM_VERSION}/graviteeio-gateway-${APIM_VERSION}/plugins/
+RUN mkdir -p ${GRAVITEEIO_HOME}/graviteeio-full-${APIM_VERSION}/graviteeio-apim-rest-api-${APIM_VERSION}/lib/ && mkdir -p ${GRAVITEEIO_HOME}/graviteeio-full-${APIM_VERSION}/graviteeio-gateway-${APIM_VERSION}/lib/ && mkdir -p ${GRAVITEEIO_HOME}/graviteeio-full-${APIM_VERSION}/graviteeio-rest-api-${APIM_VERSION}/plugins/ && mkdir -p ${GRAVITEEIO_HOME}/graviteeio-full-${APIM_VERSION}/graviteeio-gateway-${APIM_VERSION}/plugins/
```
Comment (ytvnr) [inline src/main/docker/Dockerfile-build-full-ee-v3]:
```suggestion
RUN mkdir -p ${GRAVITEEIO_HOME}/graviteeio-full-${APIM_VERSION}/graviteeio-apim-rest-api-${APIM_VERSION}/lib/ && mkdir -p ${GRAVITEEIO_HOME}/graviteeio-full-${APIM_VERSION}/graviteeio-gateway-${APIM_VERSION}/lib/ && mkdir -p ${GRAVITEEIO_HOME}/graviteeio-full-${APIM_VERSION}/graviteeio-apim-rest-api-${APIM_VERSION}/plugins/ && mkdir -p ${GRAVITEEIO_HOME}/graviteeio-full-${APIM_VERSION}/graviteeio-gateway-${APIM_VERSION}/plugins/
```

**Review by ytvnr — CHANGES_REQUESTED** [review]
I think a path has been omitted

File: src/main/docker/Dockerfile-build-full-ee-v3 (hunk 1 above)
Comment (phiz71) [reply src/main/docker/Dockerfile-build-full-ee-v3]: are you sure ? 😇  🙄

**Review by ytvnr — APPROVED** [review]
lgtm

**Pull request merged**